core/src/test/java/org/apache/calcite/test/SqlValidatorTest.java

```diff
@@ -10512,41 +10515,47 @@ private void checkCustomColumnResolving(String table) {
         + "key => descriptor(productid),\n"
         + "size => interval '1' hour))")
         .ok();
-    // negative tests.
     sql("select * from table(\n"
         + "session(\n"
-        + "^\"data\"^ => table orders,\n"
+        + "data => table orders,\n"
         + "timecol => descriptor(rowtime),\n"
-        + "key => descriptor(productid),\n"
         + "size => interval '1' hour))")
-        .fails("Param 'data' not found in function 'SESSION'; did you mean 'DATA'\\?");
```

Comment:
Seems like you have changed this test which intended to test whether the first augment is named `param` is ok? If so can you change it back?

Reply:
The test you mentioned is at L10534, I just add some new tests.

Reply:
Got it. Thanks!